### PR TITLE
Map graphics modes

### PR DESCRIPTION
**Pull request by haykam821**
This pull request maps graphics modes, which are 'fast', 'fancy', and 'fabulous'.

### CODE REVIEW

File: mappings/net/minecraft/client/options/GraphicsMode.mapping [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_5365 net/minecraft/client/options/GraphicsMode
+	FIELD field_25430 VALUES [Lnet/minecraft/class_5365;
+	FIELD field_25431 id I
+	FIELD field_25432 languageKey Ljava/lang/String;
```
Comment (liach) [inline mappings/net/minecraft/client/options/GraphicsMode.mapping]:
```suggestion
	FIELD field_25432 translationKey Ljava/lang/String;
```

File: mappings/net/minecraft/client/options/GraphicsMode.mapping [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	METHOD <init> (Ljava/lang/String;IILjava/lang/String;)V
+		ARG 3 id
+		ARG 4 languageKey
```
Comment (liach) [inline mappings/net/minecraft/client/options/GraphicsMode.mapping]:
```suggestion
		ARG 4 translationKey
```

File: mappings/net/minecraft/client/options/GraphicsMode.mapping [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	METHOD method_29591 getId ()I
+	METHOD method_29592 getModeById (I)Lnet/minecraft/class_5365;
+		ARG 0 id
+	METHOD method_29593 getLanguageKey ()Ljava/lang/String;
```
Comment (liach) [inline mappings/net/minecraft/client/options/GraphicsMode.mapping]:
```suggestion
	METHOD method_29593 getTranslationKey ()Ljava/lang/String;
```

File: mappings/net/minecraft/client/options/GraphicsMode.mapping [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+	METHOD method_29595 getNextMode ()Lnet/minecraft/class_5365;
```
Comment (liach) [inline mappings/net/minecraft/client/options/GraphicsMode.mapping]:
```suggestion
	METHOD method_29595 next ()Lnet/minecraft/class_5365;
```

File: mappings/net/minecraft/client/options/GraphicsMode.mapping [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+	METHOD method_29596 getPreviousMode ()Lnet/minecraft/class_5365;
```
Comment (liach) [inline mappings/net/minecraft/client/options/GraphicsMode.mapping]:
```suggestion
	METHOD method_29596 previous ()Lnet/minecraft/class_5365;
```

File: mappings/net/minecraft/client/options/GraphicsMode.mapping [hunk 6]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_5365 net/minecraft/client/options/GraphicsMode
+	FIELD field_25430 VALUES [Lnet/minecraft/class_5365;
+	FIELD field_25431 id I
+	FIELD field_25432 languageKey Ljava/lang/String;
+	METHOD <init> (Ljava/lang/String;IILjava/lang/String;)V
+		ARG 3 id
+		ARG 4 languageKey
+	METHOD method_29591 getId ()I
+	METHOD method_29592 getModeById (I)Lnet/minecraft/class_5365;
```
Comment (liach) [inline mappings/net/minecraft/client/options/GraphicsMode.mapping]:
```suggestion
	METHOD method_29592 byId (I)Lnet/minecraft/class_5365;
```

**Review by liach — APPROVED** [review]

**Review by Juuxel — CHANGES_REQUESTED** [review]
Should be good to merge with liach's requests.

**Comment by liach** [comment]
Mind change branch to merge into to 1.16-pre2?

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Pull request merged**